Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -432,92 +432,94 @@ class MultiTrajectory {
   }
 
   /// Retrieve a parameter proxy instance for parameters at a given index
-  /// @param parIdx Index into the parameter column
+  /// @param istate The track state
   /// @return Mutable proxy
   template <bool RO = ReadOnly, typename = std::enable_if_t<!RO>>
-  typename TrackStateProxy::Parameters parameters(IndexType parIdx) {
-    return self().parameters_impl(parIdx);
+  typename TrackStateProxy::Parameters parameters(IndexType istate) {
+    return self().parameters_impl(istate);
```

Comment:
I don't think this is the track state index, or at least it didn't use to be. This is a separate index to allow sharing of parameters + covariances.

This applies to the other parameter and covariance related functions as well.

Reply:
right I will revert these cases